public/a942d4f1c235fde42bdf1751cff43ca3.txt

```diff
@@ -0,0 +1 @@
+detectify
```

Comment:
Following [this](https://support.detectify.com/support/solutions/articles/48001061382-verification-with-txt-file?_gl=1*1bo30b2*_ga*OTAyMzQ5NjE3LjE3MDU2MDI4NDM.*_ga_F9HDKDJVKN*MTcwODQ2MTMxOS4xNi4xLjE3MDg0NjIyMDguMC4wLjA.*_fplc*ZEVSJTJCTkhqOHVkc3V0NnJ4RGl0M0Y1a2JYUnpaR083eTlHJTJGUjdNekh4dmRDQ2MzZUIlMkJrWmsyNnMlMkYlMkZKb1czZmlEJTJGN2hHeUNMbEtodDRUWFFpJTJCTWZKOThHcVlqWG5ramlNSmF3T0V6OGZ2R3pBSXpUNzBUZzkwb1RrYnglMkZqUSUzRCUzRA..) , originally tried to verify with a DNS record by adding a txt record in route 53 for demo.fileyourstatetaxes.org but since we already have a cname for that it failed. I think we will be able to remove this once it passes verification